config/database.yml.sample

```diff
@@ -15,6 +15,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  # running postgres on docker requires specifying an explicit host
+  # host: 127.0.0.1
```

Comment:
True. Also you can't specify the host as `localhost` or it will switch to sockets instead of TCP